powertools-core/src/main/resources/version.properties

```diff
@@ -0,0 +1 @@
+powertools.version=${project.version}
```

Comment:
Does the value come from `pom.xml` during runtime?

Reply:
[this](https://github.com/eldimi/aws-lambda-powertools-java/blob/chore/user_agent_sdk_clients/powertools-core/pom.xml#L101) should do the trick. It is already tested in the unit tests. I'll import the library in a project to test again and make sure I didn't miss anything

Reply:
Probably worth publishing locally and then unzipping the jar and making sure that it contains `1.17.0-SNAPSHOT` in there

Reply:
> Probably worth publishing locally and then unzipping the jar and making sure that it contains `1.17.0-SNAPSHOT` in there

I scratched my own itch and checked this from your fork - looks good - here's version.properties in the core jar:

```
# The filtered properties can have variables that are filled in by system properties or project properties.
# See https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html
#
# The values are replaced before copying the resources to the main output directory. Therefore, as soon as the build phase is completed,
# the values should have been replaced if the properties are available and if 'filtering' is set to true in the pom.xml
#
#
# The 1.17.0-SNAPSHOT is retrieved from the respective pom.xml property
powertools.version=1.17.0-SNAPSHOT
```